src/LogDensityProblems.jl

```diff
@@ -12,14 +12,13 @@ documentation.
 """
 module LogDensityProblems
 
-export TransformedLogDensity, ADgradient
+export ADgradient
 
 using ArgCheck: @argcheck
 using DocStringExtensions: SIGNATURES, TYPEDEF
-using UnPack: @unpack
-import Random
+using Random: AbstractRNG, default_rng
```

Comment:
`default_rng` requires Julia >= 1.3, so one would have to bump the Julia compat - or keep using `GLOBAL_RNG`.